ci/lint/06_script.sh

```diff
@@ -6,8 +6,8 @@
 
 export LC_ALL=C
 
-if [ "$TRAVIS_EVENT_TYPE" = "pull_request" ]; then
-  test/lint/commit-script-check.sh $TRAVIS_COMMIT_RANGE
+if [ "$CI_EVENT_TYPE" = "pull_request" ]; then
```

Comment:
```suggestion
if [ "$EVENT_TYPE" = "pull_request" ]; then
```
or (remove the env this env var in the workflow)
```suggestion
if [ "$GITHUB_EVENT_TYPE" = "pull_request" ]; then
```
If you want to preserve the `CI_EVENT_TYPE`, you could edit workflow file to change the env name.